src/conf/startup.properties

```diff
@@ -136,6 +136,12 @@ prism.configstore.listeners=org.apache.falcon.entity.v0.EntityGraph,\
 
 *.falcon.cleanup.service.frequency=days(1)
 
+# Default number of threads to be used to restore entities.
```

Comment:
Can we follow the convention we have followed for other variables of config store, such as *.config.store.uri. 
These two could be : 
*.config.store.num.threads.load.entities
*.config.store.start.timeout.minutes